Backend/src/main/java/com/qelem/api/model/UserModel.java

```diff
@@ -65,6 +65,7 @@ public enum ROLE {
     @Exclude
     private List<AnswerVote> answerVotes;
 
+    // TODO: needs to be changed as the new storage method doesn't follow the same pattern
```

Comment:
I was tired. And it was important and shouldn't be noted. And it is kind of a standard thing Todo's

Reply:
The todo was referring to creating a link for the photos which we would be able to call like any other api's. The issue was not part of this, just to mention it. This was so that it wouldn't be forgotten on the next go of API improvements. And one more thing, try to be polite in you reviews. This feels more like a personal attack than a code review. Seriously, not cool.

Reply:
The todo was referring to creating a link for the photos which we would be able to call like other api's. The issue was not part of this, just to mention it. This was so that it wouldn't be forgotten on the next go of API improvements. And 'do your job man'? not cool. This is not even this pull request's issue concern.

Reply:
@hello-world-was-taken It actually is this pr's concern, how are you gonna have a profile network service with out a profile api?

Reply:
@brukted  The profile api works just fine. The todo was only for providing actual links for the images. When I say actual links, I mean the ones where the app will be able to show.

Reply:
@brukted why are we still talking about sth we have already discussed in person? Feels weird